packages/@aws-cdk/core/lib/bundling.ts

```diff
@@ -110,6 +110,7 @@ export class BundlingDockerImage {
 
```

Comment:
@jogold does it make sense to rename this to `DockerImage`? What's "bundling" about this?

Reply:
It's the docker image used when bundling an asset/accepted in the `bundling` options... but I agree that there's nothing specific to bundling in its implementation.

Reply:
Let's leave this for a future change